postgres/setup.sh

```diff
@@ -0,0 +1,57 @@
+#! /bin/bash
+
+set -e
+
+sed -i "/host all all all/d" "$PGDATA/pg_hba.conf"
+echo "host all $WUBLOADER_USER all md5" >> "$PGDATA/pg_hba.conf"
+
+echo "Creating $WUBLOADER_USER"
+psql -v ON_ERROR_STOP=1 -U postgres <<-EOSQL
+
+CREATE USER $WUBLOADER_USER LOGIN PASSWORD '$WUBLOADER_PASSWORD';
+
+EOSQL
+
+
+if [ -n "$REPLICATION_USER" ]; then
+	echo "Creating $REPLICATION_USER"
+	echo "host replication $REPLICATION_USER all md5" >> "$PGDATA/pg_hba.conf"
+	psql -v ON_ERROR_STOP=1 -U postgres <<-EOSQL
+
+	CREATE USER $REPLICATION_USER LOGIN REPLICATION PASSWORD '$REPLICATION_PASSWORD';
+
+	EOSQL
+
+	cat >> ${PGDATA}/postgresql.conf <<-EOF
+	wal_level = replica
+	archive_mode = on
+	archive_command = 'cd .'
+	max_wal_senders = 8
+	wal_keep_segments = 8
+	EOF
+
+fi
+
+if [ -a /mnt/wubloader/nodes.csv ]; then
+	echo "Loading nodes from nodes.csv"
+	psql -v -U postgres -d ${POSTGRES_DB} <<-EOF
+	CREATE TABLE IF NOT EXISTS nodes (
+		name TEXT PRIMARY KEY,
+		url TEXT NOT NULL,
+		backfill_from BOOLEAN NOT NULL DEFAULT TRUE);
```

Comment:
I'm concerned that these schemas are now written in two places. They could get out of sync, and there'd be no error.
Maybe it makes more sense to remove the schema stuff from common.database and put it entirely here?

Reply:
Agreed but I'll do the transfer after merging this to master.